src/main/java/org/folio/converter/ParsedRecordToQuickMarcConverter.java

```diff
@@ -2,82 +2,76 @@
 
 import static org.folio.converter.StringConstants.BLVL;
 import static org.folio.converter.StringConstants.CONTENT;
+import static org.folio.converter.StringConstants.EMPTY_STRING;
+import static org.folio.converter.StringConstants.SPACE;
+import static org.folio.converter.StringConstants.TAG_008;
 import static org.folio.converter.StringConstants.TYPE;
 
 import io.vertx.core.json.JsonObject;
 import org.apache.commons.io.IOUtils;
 import org.folio.rest.jaxrs.model.Field;
 import org.folio.rest.jaxrs.model.QuickMarcJson;
 import org.folio.srs.model.ParsedRecord;
+import org.jetbrains.annotations.NotNull;
 import org.marc4j.MarcJsonReader;
-import org.marc4j.MarcReader;
 import org.marc4j.marc.DataField;
+import org.marc4j.marc.Leader;
 import org.marc4j.marc.Record;
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.stereotype.Component;
 
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.LinkedHashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 @Component
 public class ParsedRecordToQuickMarcConverter implements Converter<ParsedRecord, QuickMarcJson> {
   @Override
-  public QuickMarcJson convert(ParsedRecord parsedRecord) {
+  public QuickMarcJson convert(@NotNull ParsedRecord parsedRecord) {
```

Comment:
Jetbrains annotation? Do we really need it?

Reply:
Not at all, I forgot to remove it